### PR TITLE
Changes for June 7, 2023

### PR DESCRIPTION
**Pull request by adderall-prozac**
No additional functionality to the API. Just additional symbol permissions possible on the exchange.

### CODE REVIEW

**Review by aisling-2 — APPROVED** [review]

File: rest-api.md [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (LottisPottispoo) [inline rest-api.md:67]: Update

File: rest-api_CN.md [hunk 2]
```diff
@@ -514,7 +519,7 @@ GET /api/v3/exchangeInfo
 * 所有的参数都是可选的.
 * `permissions` 支持单个或者多个值, 比如 `SPOT`, `["MARGIN","LEVERAGED"]`.
 * 如果`permissions`值没有提供, 其默认值为 `["SPOT","MARGIN","LEVERAGED"]`.
-  * 如果想取接口 `GET /api/v3/exchangeInfo` 的所有交易对, 则需要设置此参数的所有可能交易权限值, 比如 `permissions=["SPOT","MARGIN","LEVERAGED","TRD_GRP_002","TRD_GRP_003","TRD_GRP_004","TRD_GRP_005","TRD_GRP_006","TRD_GRP_007"])`
+  * 如果想取接口 `GET /api/v3/exchangeInfo` 的所有交易对, 则需要设置此参数的所有可能交易权限值, 比如 `permissions=["SPOT","MARGIN","LEVERAGED","TRD_GRP_002","TRD_GRP_003","TRD_GRP_004","TRD_GRP_005","TRD_GRP_006","TRD_GRP_007","TRD_GRP_008","TRD_GRP_009","TRD_GRP_010","TRD_GRP_011","TRD_GRP_012","TRD_GRP_013"]`)
```
Comment (LottisPottispoo) [inline rest-api_CN.md:522]: Get

File: web-socket-api.md [hunk 3]
```diff
@@ -66,7 +66,7 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Public WebSocket API for Binance (2023-03-13)
+# Public WebSocket API for Binance (2023-06-07)
```
Comment (LottisPottispoo) [inline web-socket-api.md:70]: Update this for me

**Review by LottisPottispoo — COMMENTED** [review]
Order update for me

**Pull request merged**